specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/AppServiceEnvironments.json

```diff
@@ -1209,6 +1259,56 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/hostingEnvironments/{name}/outboundNetworkDependenciesEndpoints": {
+      "get": {
+        "tags": [
+          "AppServiceEnvironments"
+        ],
+        "summary": "Get the network endpoints of all outbound dependencies of an App Service Environment.",
+        "description": "Get the network endpoints of all outbound dependencies of an App Service Environment.",
+        "operationId": "AppServiceEnvironments_GetOutboundNetworkDependenciesEndpoints",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Name of the App Service Environment.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/OutboundEnvironmentEndpoint"
+              }
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Outbound Network Dependencies Endpoints": {
+            "$ref": "./examples/GetOutboundNetworkDependenciesEndpoints.json"
```

Comment:
The example uses paging, but this has not been called out here with an "x-ms-pageable" declaration. Please can you either update the example, or the schema, depending on whether or not you are using paging on this response?